src/app/header/header.component.html

```diff
@@ -1,8 +1,11 @@
 <header>
-    <mat-toolbar color="primary">
-        <mat-toolbar-row>
-            <button mat-button [routerLink]="['/dtw']" routerLinkActive="active">Dynamic Time Warping</button>
-            <button mat-button [routerLink]="['/edit-distance']" routerLinkActive="active">Edit Distance</button>
-        </mat-toolbar-row>
-    </mat-toolbar>
+  <mat-toolbar color="primary">
+    <mat-toolbar-row>
+      <h1 class="title">DTW Project</h1>
```

Comment:
I think it is better to name "Algorithms Visualisation" instead of DTW Project, since we are not implementing only DTW.

Reply:
Fixed